### DISCUSSION TITLE
Figure of brain map showing included sulci

### DISCUSSION THREAD

**Discussion by Ssiimonn** [brainvisa/brainvisa.github.io#166] [Q&A]
Hi all,

First of all, a thank you to the developers for their awesome work on BrainVISA.

I've successfully run the morphologist pipeline and obtained the sulci length, depth, width and surface for the brainVISA ROIs.

As expected, a portion of sulcal measurements fail for some scans. We therefore set an arbitrary threshold to include only sulci that have a successful measurement for 90%+ of the scans.

Now, in our paper, I'd like to show a figure of a template brain showing which sulci were included to our final analyses after this threshold was applied. Can someone help me in how to obtain such a figure?

Kind regards,
Simon

**Comment by denisri** [comment]
Hi,
You mean a figure showing the sulci of the nomenclature ? You can find one here:
https://brainvisa.info/web/morphologist.html
(on in the paper Borne 2020, Media)
If you need to make a custom one for the sulci you have actually used, perhaps you can start from this figure and grey out sulci you are not using, or highlight the ones you are using...?
Best,
Denis

**Reply by Ssiimonn** [reply]
Hi Denis,

Thank you for your reply.

I guess I'll have to put my vector image editing skills to the test, then ;)

Cheers,
Simon
